### PR TITLE
feat: insert generated benchmark data into db, for next visualization

### PR DESCRIPTION
**Pull request by chanhsu001**
here left 3 things to solve:
1) python script could read "BENCH_DB_CONN" variable (set in repo's secrets)
2) apt update and apt install python support lib could be OK.
3) i'm not sure if insert_benchmark_to_db could be called.

### CODE REVIEW

File: ckb-bench/devtools/ci/ansible/playbook.yml [hunk 1]
```diff
@@ -69,3 +69,25 @@
...
+      become: true
+      become_method: sudo
+      apt:
+        name: ['libpq-dev', 'python3-psycopg2']
+        state: latest
+        install_recommends: yes
+        update_cache: yes
+      tags:
+        - apt_install_python_packages
+
```
Comment (keroro520) [inline ckb-bench/devtools/ci/ansible/playbook.yml:93]: As we can access the test result from the hosted machine, we do not need to write the database from the remote machine via Ansible.

IMO, it should be done in `benchmark.sh`.

File: ckb-bench/devtools/ci/ansible/files/save_bench_to_db.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import os
+import sys
+import datetime;
+from urllib.parse import urlparse
+
+import psycopg2
+import yaml
+from psycopg2 import sql
+
+BENCHMARK_POSTGRES_SECRET = os.getenv("BENCH_DB_CONN")
```
Comment (keroro520) [inline ckb-bench/devtools/ci/ansible/files/save_bench_to_db.py:14]: Use the same name.

File: ckb-bench/devtools/ci/ansible/files/save_bench_to_db.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                    n_inout, n_nodes, to_block_number,
+                    total_transactions, total_transactions_size, transactions_per_second,
+                    transactions_size_per_second) values (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)"""),
+                [datetime.datetime.now(), metric["average_block_time_ms"], metric["average_block_transactions"],
+                 metric["average_block_transactions_size"], metric["ckb_version"], metric["delay_time_ms"],
+                 metric["from_block_number"], metric["instance_bastion_type"], metric["instance_type"],
+                 metric["n_inout"],
+                 metric["n_nodes"], metric["to_block_number"], metric["total_transactions"],
+                 metric["total_transactions_size"], metric["transactions_per_second"],
+                 metric["transactions_size_per_second"]])
```
Comment (keroro520) [inline ckb-bench/devtools/ci/ansible/files/save_bench_to_db.py:59]: is there schema file?

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: ckb-bench/devtools/ci/ansible/playbook.yml (hunk 1 above)
Comment (liya2017) [reply ckb-bench/devtools/ci/ansible/playbook.yml:93]: can parsing the `ckb-bench.brief.md` directly   if do it in benchmark.sh, have already copy it into `GITHUB_WORKSPACE`.

**Pull request closed without merging**